content/html/concepts/elements/terms/applet/applet.md

```diff
@@ -16,6 +16,11 @@ CatalogContent:
 
 The **`<applet>`** element was used to embed Java applets into an HTML page. It has since been deprecated and removed from the [HTML specification](https://html.spec.whatwg.org/). This tag should no longer be used as browsers no longer support it.
 
+In the following picture we can see an example of an Java applet:
+
+<img scr="/Users/evadome/Desktop/EVA DOME/Codecademy/Docs Github/docs/media/Applet Lifecycle.png" alt="Lifecycle Image"/>
```

Comment:
This is not the right way to add the media. You need to fix the link. This dir won't work. Make it like this `https://raw.githubusercontent.com/Codecademy/docs/main/media/table-example-1.png`. Also rename you image name in small letters and instead of space use `-`.